docs/dictionary/en-custom.txt

```diff
@@ -474,6 +474,7 @@ subnet
 subnets
 sudo
 sudoers
+supercedent
```

Comment:
it seems `supersede` is the correct spelling?
```
Supersedent: This term is not commonly used in English and is often considered incorrect. It is sometimes mistakenly used in place of "supersede."
```

Reply:
It's in reference to this Zuul concept: https://zuul-ci.org/docs/zuul/latest/config/pipeline.html#value-pipeline.manager.supercedent

Reply:
"yay" :).